pom.xml

```diff
@@ -119,25 +119,28 @@
     <dependencyManagement>
         <dependencies>
             <dependency>
-                <groupId>javax.enterprise</groupId>
-                <artifactId>cdi-api</artifactId>
+                <groupId>jakarta.enterprise</groupId>
+                <artifactId>jakarta.enterprise.cdi-api</artifactId>
                 <version>${cdi-api.version}</version>
+                <scope>provided</scope>
                 <exclusions>
                     <exclusion>
```

Comment:
Based on other threads I've seen, does this also need an EJB exclude as it was added by mistake to Jakarta CDI?

Reply:
@kenfinnigan yes, thanks for pointing it out.